src/bash/sdv-health

```diff
@@ -247,13 +221,38 @@ check_cloudconnector_deviceid()
 	fi
 }
 
+cm_get_container_id ()
+{
+	local CONT_NAME=$1
+	${KANTO_CMD} get -n $CONT_NAME 2>/dev/null | jq -r " .container_id" 
+}
+
+cm_check_logging_type ()
+{
+	local CONT_ID=$1
+	${KANTO_CMD} get -n databroker | jq -r " .host_config.log_config.driver_config.type"
```

Comment:
CONT_ID is not used and container name is hardcoded, please fix.

Reply:
Thanks, mistake fixed.